README.md

```diff
@@ -1,5 +1,5 @@
 <p>
-  <img src="https://cdn.rawgit.com/MessageKit/MessageKit/master/Assets/mklogo.svg" title="MessageKit logo">
+  <img src="https://raw.githubusercontent.com/timglorioso/MessageKit/master/Assets/messagekit-logo.png" title="MessageKit logo">
```

Comment:
@timglorioso Can we change this back and I'll merge the PR?

Reply:
Assuming you want the URL updated to reflect the new filename?

Reply:
You could also just rename the file to match the old one 😃